src/sass/_base.scss

```diff
@@ -0,0 +1,146 @@
+html {
+    box-sizing: border-box;
+}
+
+*,
+*::before,
+*::after {
+    box-sizing: inherit;
+}
+
+body {
+    color: $color-text-body;
+    font-family: "Commissioner", sans-serif;
+    font-weight: 400;
+    margin: 0;
+    height: 100vh;
+}
+
+.sidenav {
+  background-color: $color-grey;
+  position: fixed;
+  height: 100%;
+  width: 432px;
+  top: 0;
+  left: 0;
+  color: $color-offwhite;
+  padding: 15px 35px;
+  
+  ul {
+    padding: 0;
+  }
+  
+  li {
+    list-style-type: none;
+    text-align: center;
+    margin: 15% 0;
+    
+    a {
+      color: $color-offwhite;
+      text-decoration: none;
+      font-size: 2em;
+      font-weight: 100;      
+    }  
+  }
+  
+  .social { 
+    img {
+      width: 32%;
+    }
+  }
+}
+
+main {
+    flex: 1 0 auto;
```

Comment:
This has no effect because the parent of the `main` element is not `display: flex;`
```suggestion
```